.github/workflows/cypress-third-party.yml

```diff
@@ -0,0 +1,132 @@
+name: CI - Cypress (e2e) 3rd party donation tests
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build-client:
+    name: Build
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout Source Files
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # tag=v3
+
+      - name: Checkout client-config
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # tag=v3
+        with:
+          repository: freeCodeCamp/client-config
+          path: client-config
+
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@2fddd8803e2f5c9604345a0b591c3020ee971a93 # tag=v3
+        with:
+          node-version: 16.x
+          cache: 'npm'
+
+      - name: Set freeCodeCamp Environment Variables
+        run: |
+          sed '/STRIPE/d; /PAYPAL/d;' sample.env > .env
+          echo 'STRIPE_PUBLIC_KEY=${{ secrets.STRIPE_PUBLIC_KEY }}' >> .env
+          echo 'STRIPE_SECRET_KEY=${{ secrets.STRIPE_SECRET_KEY }}' >> .env
+          echo 'PAYPAL_CLIENT_ID=${{ secrets.PAYPAL_CLIENT_ID }}' >> .env
+          echo 'PAYPAL_SECRET=${{ secrets.PAYPAL_SECRET }}' >> .env
+
+      - name: Install and Build
+        run: |
+          npm ci
+          npm run build
+
+      - name: Move serve.json to Public Folder
+        run: cp client-config/serve.json client/public/serve.json
+
+      # We tar them for performance reasons - uploading a lot of files is slow.
+      - name: Tar Files
+        run: tar -cf client-artifact.tar client/public
+
+      - name: Upload Client Artifact
+        uses: actions/upload-artifact@3cea5372237819ed00197afe530f5a7ea3e805c8 # tag=v3
+        with:
+          name: client-artifact
+          path: client-artifact.tar
+
+      - name: Upload Webpack Stats
+        uses: actions/upload-artifact@3cea5372237819ed00197afe530f5a7ea3e805c8 # tag=v3
+        with:
+          name: webpack-stats
+          path: client/public/stats.json
+
+  cypress-run:
+    name: Test
+    runs-on: ubuntu-20.04
+    needs: build-client
+    strategy:
+      fail-fast: false
+      matrix:
+        browsers: [chrome, electron]
```

Comment:
Just to confirm, do we want to run the donation tests on electron?

Reply:
Good catch, I don't think there's any need to.

Reply:
Ok so if we are going to use Chrome only, how about removing the strategy, the artifact upload and download all-together similar to the last commit?